cookbooks/bcpc-hadoop/recipes/zookeeper_impl.rb

```diff
@@ -25,6 +25,11 @@
   to "/usr/hdp/current/zookeeper-client/bin/zookeeper-server-initialize"
 end
 
+#Install jolokia's jvm agent to node['bcpc']['jolokia']['path']
+if node[:bcpc][:jolokia][:enable] == true
+  include_recipe 'bcpc-hadoop::jolokia'
```

Comment:
This looks duplicated with [bcpc_kafka::zookeeper_server](https://github.com/bloomberg/chef-bach/blob/8952764c95cf77063f2303a0b08231a4599fd1a3/cookbooks/bcpc_kafka/recipes/zookeeper_server.rb#L18)?

Reply:
Thanks @cbaenziger . Updated [bcpc_kafka::zookeeper_server](https://github.com/bloomberg/chef-bach/blob/8952764c95cf77063f2303a0b08231a4599fd1a3/cookbooks/bcpc_kafka/recipes/zookeeper_server.rb#L17)